client/request_types.go

```diff
@@ -10,6 +10,7 @@
 package client
```

Comment:
Why does this commit have all these changes?
@kingpinXD are you sure your pr is right?